### PR TITLE
create db_build.js for building database

### PR DESCRIPTION
**Pull request by iPhatty**
Relates #2 & #3

### CODE REVIEW

File: src/model/database/db_build.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const { QueryFile } = require('pg-promise');
+const db = require('./db_connection');
+
+// used to execute a database build file
+const sql = file => QueryFile(path.join(__dirname, file), { minify: true });
```
Comment (haydnba) [inline src/model/database/db_build.js:6]: does this parameter `file`need to be in parentheses to satisfy ESlint??

**Review by haydnba — COMMENTED** [review]
see my inline comment

File: src/model/database/db_build.js (hunk 1 above)
Comment (haydnba) [reply src/model/database/db_build.js:6]: actually - this is not an issue - AirB&B only cares with new lines!

Comment (iPhatty) [reply src/model/database/db_build.js:6]: Don't worry buddy, eslint is ok with it  🍚 👍

File: src/model/database/db_build.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+const build = sql('./db_build.sql');
+
+// database building for test
+const runDbBuild = (cb) => {
+  db.query(build)
+    .then((res) => {
+      console.log('res: ', res);
+      cb();
```
Comment (i2xzy) [inline src/model/database/db_build.js]: pass the res into the callback instead of logging it

File: src/model/database/db_build.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    })
+    .catch(e => console.error('error: ', e));
```
Comment (i2xzy) [inline src/model/database/db_build.js]: pass the err into a callback instead of logging it

**Review by haydnba — APPROVED** [review]
Lovely 🌷

**Review by n1c0z — APPROVED** [review]
:1st_place_medal: :+1:  :smile_cat:

**Pull request merged**